src/coreimage.cs

```diff
@@ -1328,6 +1351,18 @@ interface CIImageInitializationOptionsKeys {
 		[TV (12, 0), iOS (12, 0), Mac (10, 14)]
 		[Field ("kCIImageAuxiliaryPortraitEffectsMatte")]
 		NSString AuxiliaryPortraitEffectsMatteKey { get; }
+
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCIImageAuxiliarySemanticSegmentationSkinMatte")]
+		NSString AuxiliarySemanticSegmentationSkinMatteKey { get; }
+
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCIImageAuxiliarySemanticSegmentationHairMatte")]
+		NSString AuxiliarySemanticSegmentationHairMatteKey { get; }
+
+		[iOS (13,0)][TV (13,0)][Watch (6,0)][Mac (10,15)]
+		[Field ("kCIImageAuxiliarySemanticSegmentationTeethMatte")]
+		NSString AuxiliarySemanticSegmentationTeethMatteKey { get; }
```

Comment:
This look like they could be a smart enum.

Reply:
it's actually a inside a type used tp create a strong dictionary `[StrongDictionary ("CIImageInitializationOptionsKeys")]` but I did not add the keys to it :| I'll fix this

Makes me wish for https://github.com/xamarin/xamarin-macios/issues/6785 even more